packages/braintree-integration/src/braintree-paypal-ach/braintree-paypal-ach-payment-strategy.ts

```diff
@@ -79,16 +79,18 @@ export default class BraintreePaypalAchPaymentStrategy implements PaymentStrateg
             throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
         }
 
-        if (!this.mandateText) {
+        const mandateText = this.getMandateText && this.getMandateText();
```

Comment:
It might be better to check this.getMandateText type. It should be a function, otherwise, this line of code will have an error.